src/components/PrimaryPanes/Outline.js

```diff
@@ -61,20 +61,25 @@ export class Outline extends Component {
   renderFunctions(symbols: Array<SymbolDeclaration>) {
     return (
       <ul className="outline-list">
-        { symbols.map(func => this.renderFunction(func)) }
+        {symbols.map(func => this.renderFunction(func))}
       </ul>
     );
   }
 
   render() {
     const { isHidden, symbols } = this.props;
 
-    const symbolsToDisplay = symbols.functions
-      .filter(func => func.name != "anonymous");
+    const symbolsToDisplay = symbols.functions.filter(
+      func => func.name != "anonymous"
+    );
 
     return (
       <div className={classnames("outline", { hidden: isHidden })}>
-        { symbolsToDisplay.length > 0 ? this.renderFunctions(symbolsToDisplay) : this.renderPlaceholder() }
+        {symbolsToDisplay.length > 0 ? (
+          this.renderFunctions(symbolsToDisplay)
+        ) : (
+          this.renderPlaceholder()
+        )}
       </div>
     );
   }
```

Comment:
@Anzumana The change to this file seems to be pulled in from somewhere else

Reply:
i merged master into my branch and this is the current state 
https://github.com/devtools-html/debugger.html/blob/master/src/components/PrimaryPanes/Outline.js
not sure why still showing the diff.